src/v2/view-builder/views/ov/EnrollOVWithEmailChannelView.js

```diff
@@ -0,0 +1,23 @@
+import { loc } from 'okta';
+import BaseForm from '../../internals/BaseForm';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorEnrollFooter from '../../components/AuthenticatorEnrollFooter';
+
+const Body = BaseForm.extend({
+  title: loc('oie.enroll.okta_verify.enroll.channel.email.title', 'login'),
+  save: loc('oie.enroll.okta_verify.setupLink', 'login'),
+  getUISchema () {
+    const schemas = BaseForm.prototype.getUISchema.apply(this, arguments);
+    const description = {
+      View: loc('oie.enroll.okta_verify.select.channel.email.description', 'login'),
```

Comment:
can this be `explain` of the input option?

Reply:
this is description is at bottom of the ui schema which looks like it shall be explain. and people tends to ask same question when they look at the code.
I guess the good part is we dont look into uiSchemas. I'll slightly prefer using explain otherwise add comments in the code.